FabNESO/read_outputs.py

```diff
@@ -0,0 +1,21 @@
+"""A script to read the outputs of the NESO solvers."""
```

Comment:
```suggestion
"""Functions for reading the outputs of the NESO solvers."""
```

As far as I can tell this is intended for use as a library module not as a script?